### PR TITLE
Newly added TC_Loans_PersonalLoan_001.java

### PR DESCRIPTION
**Pull request by SumanthELF**
I have added one new automation script under Loans, personal loanm

### CODE REVIEW

File: Selenium.Project/src/loans/TC_Loans_PersonalLoan_001.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package loans;
+
```
Comment (dineshavenger09) [inline Selenium.Project/src/loans/TC_Loans_PersonalLoan_001.java:2]: add author name

Comment (dineshavenger09) [reply Selenium.Project/src/loans/TC_Loans_PersonalLoan_001.java:2]: Please follow all the standards and indent your script properly and then ask for money

**Comment by dineshavenger09** [comment]
Fix all the comments

File: Selenium.Project/src/loans/TC_Loans_PersonalLoan_001.java (hunk 1 above)
Comment (SumanthELF) [reply Selenium.Project/src/loans/TC_Loans_PersonalLoan_001.java:2]: Please check it is properly indented and i have followed all the standards

**Pull request merged**